lib/propolis/src/block/mod.rs

```diff
@@ -38,6 +38,9 @@ pub type ByteLen = usize;
 /// is not choosing a block size, a default of 512B is used.
 pub const DEFAULT_BLOCK_SIZE: u32 = 512;
 
+// XXX: completely arb for now
+pub const MAX_FILE_WORKERS: usize = 32;
```

Comment:
nit: I don't know why this needs to live in propolis-lib, as there's nothing in the underlying implementation which really "cares" about the upper bound on the worker threads.  We can probably just let proplis-server (and standalone) define its own upper limit?

Reply:
I've changed things so propolis-server and propolis-standalone use their own max (which happens to be the same currently).